nvflare/private/fed/client/client_executor.py

```diff
@@ -141,7 +135,7 @@ def send_aux_command(self, shareable: Shareable, job_id):
 
     def cleanup(self):
         """Cleanup."""
-        self.pipe.clear()
+        pass
```

Comment:
Just a note that part of the cleanup of this to the dev branch is in PR: https://github.com/NVIDIA/NVFlare/pull/761

Reply:
Once merged with dev, this will be addressed with other PR.